pkg/parser/qs/tojson.go

```diff
@@ -0,0 +1,15 @@
+package qs
+
+import (
+	"github.com/joncalhoun/qson"
```

Comment:
kenapa pakai lib yang sudah di archive mas?
kalau coba pakai https://github.com/gorilla/schema gmana?

Reply:
belum nemu library yg lebih flexible dari qson. Kalau gorilla schema, dia parse data dari form (body) value kan ya? sementara kita butuhnya dari query params, misal ada call ke API

http://localhost:8000/books?price[gte]=20000&price[lte]=50000

nah itu nanti akan dipetakan dan jadi SQL sbb:

SELECT *
FROM books
WHERE price >= 20000 AND price <= 50000

Reply:
@ilmimris gak jadi pake qson, krn ada bug ketika valuenya pake %25. Jadi skrg dah diganti ke https://github.com/derekstavis/go-qs